### ISSUE TITLE
HEALPix Utils function

### ISSUE THREAD

**Issue by tech3371** [IMAP-Science-Operations-Center/imap_processing#994]
some links:
https://healpy.readthedocs.io/en/latest/install.html
https://healpix.sourceforge.io/

**Comment by tmplummer** [comment]
It is not clear to me that we actually need to implement this. If the L2 code was modularized to the extent that any binning routine could be plugged in, it would allow whoever develops the Mapper tool to just plug in a binning function that would bin to HEALPix rather than Lat/Lon grid. It my be moot since Ultra is now requesting HEALPix number be included in their pset product.

**Comment by tech3371** [comment]
I believe you guys completed this and not me. That's why I assigned you guys.

**Comment by nkerman** [comment]
Yes, I think it's fair to close this, at least from the Ultra Standpoint. @subagonsouth do you have any objection to closing this?

**Comment by tmplummer** [comment]
Agree. Closing as duplicate.
